### PR TITLE
Added ability to specify a function name for the callback

### PR DESCRIPTION
**Pull request by rapind**
Personally need it to leverage jsonp caching, but should also solve #14

### CODE REVIEW

**Comment by roderickhsiao** [comment]
any update for this PR ?
@TooTallNate

**Comment by TooTallNate** [comment]
Seems mostly fine to me. The option name `cbName` I don't love so much.

/cc @rauchg thoughts?

**Comment by rauchg** [comment]
How about `name` ?

**Comment by rapind** [comment]
No preference here. Happy with w/e.

**Comment by roderickhsiao** [comment]
as we are using `opts.prefix` for callback prefix (not `cbPrefix`), 
guess its good to just use `name` 

How do you guys think?

**Comment by roderickhsiao** [comment]
@TooTallNate @rauchg do you think the PR looks good?

File: Readme.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ $ bower install jsonp
     disable (defaults to `60000`)
   - `prefix` (`String`) prefix for the global callback functions that
     handle jsonp responses (defaults to `__jp`)
+  - `cbName` (`String`) name of the global callback functions that
```
Comment (feross) [inline Readme.md]: Forgot to update this to `name`.

File: test/basic.js [hunk 2]
```diff
@@ -28,3 +28,16 @@ test('timeout', function (t) {
...
 });
+
+test('named callback', function (t) {
+  t.plan(1);
+  var obj = {
+    beep: 'boop',
+    yo: 'dawg'
+  };
+  var q = querystring.encode(obj);
+  jsonp(ENDPOINT + '?' + q, { cbName: 'namedCb' }, function (err, data) {
```
Comment (feross) [inline test/basic.js]: Test needs to be updated. It's failing.

**Comment by feross** [comment]
PR still needs work. See inline comments.

**Comment by roderickhsiao** [comment]
@feross @rapind  Thanks!

**Comment by roderickhsiao** [comment]
@TooTallNate @rauchg does the PR looks good for you guys :)

**Comment by feross** [comment]
LGTM

**Comment by roderickhsiao** [comment]
@TooTallNate @rauchg mind to take a look :+1: ? thanks

**Comment by feross** [comment]
@TooTallNate, @rauchg - if you add me as contributor/owner, I can take care of this release.

**Comment by rauchg** [comment]
`0.2.0` out! Thanks. Also pinned the `debug` dep

**Comment by roderickhsiao** [comment]
Thanks all!

**Comment by rapind** [comment]
@roderickhsiao you're like an Open Source project management god. Keep up the good work man!

**Comment by rauchg** [comment]
🙏

**Pull request merged**